content/docs/guides/job-scheduling.md

```diff
@@ -0,0 +1,101 @@
++++
+title = "Job Scheduling"
```

Comment:
To help people find this page, I'd recommend adding a section at the end of the TFjob page:
https://www.kubeflow.org/docs/guides/components/tftraining/

Something like this:

## More information

* See how to [add gang scheduling to a job](/docs/guides/job-scheduling).

Reply:
Thanks for your advices!
I have made some change, please have a look again, thanks!